binchihua/test/common/list_test.c

```diff
@@ -0,0 +1,273 @@
+#include "../../src/common/list.h"
+#include <stdio.h>
+
+#define VOID_0 ((void) 0)
```

Comment:
These definitions (`VOID_0`, `ELEMENTSOF` and `assert_se`) are currently used only here. As soon as there are other use cases, we can move them to a proper location.

Reply:
Sounds good.